data/modelCuration/KEGG_pathways_maping.tsv

```diff
@@ -0,0 +1,4059 @@
+metabolism	KEGG pathways
+pyruvate metabolism	sce00620
```

Comment:
Are these KEGG pathways annotations that should be annotated as `kegg.pathway`? Know that annotations can actually have multiple entries, so we can assign reactions with multiple KEGG pathways. Perhaps a strategy to gather this, is to look at the reaction-specific `kegg.reaction` annotations and match those to the KEGG pathways they are involved in?

Reply:
I actually just see that many reactions are already have `kegg.pathway` annotations. Maybe not all, so good still be good to check. Can be in separare Issue/PR though.

Reply:
I agree with that. One reaction may belongs to multiple KEGG pathways. And we can solve that in in separare Issue/PR